doc/api/process.md

```diff
@@ -373,7 +373,10 @@ It is important to take note of the following:
 * `SIGTERM` is not supported on Windows, it can be listened on.
 * `SIGINT` from the terminal is supported on all platforms, and can usually be
   generated with `CTRL+C` (though this may be configurable). It is not generated
-  when terminal raw mode is enabled.
+  when terminal raw mode is enabled. If your program listens for `SIGINT`, it is
```

Comment:
Please avoid the use of informal pronouns like `you` or `your` in documentation. This can be rewritten as: `If a program listens for 'SIGINT'...`